src/github.rs

```diff
@@ -1,3 +1,5 @@
+//! This module implements functionality for interacting with github.
```

Comment:
*GitHub, not github :)

Reply:
I'd hate to burn CI time on this.  @carols10cents do you know if the issue with [no-ci] on bors-voyager has been fixed?  I know last time I used that it froze something in the pipeline and you had to override it.